### DISCUSSION TITLE
[iid] FirebaseInstanceId.getId() method is not available in plugin

### DISCUSSION THREAD

**Discussion by talent-apps** [firebase/flutterfire#5584] [Feature request]
If a Firebase user uses **multiple** devices, there is a separate token for each one of his devices.
In order to identify the device, the following method needs to be available in the plugin:

[FirebaseInstanceId.getId()](https://firebase.google.com/docs/reference/android/com/google/firebase/iid/FirebaseInstanceId.html#getId())

For more info, see:
https://stackoverflow.com/questions/44496966/how-to-handle-multiple-firebase-fcm-tokens-per-user

**Comment by ghost** [comment]
I have a pull request opened for this: #2328
